modules/ROOT/pages/introduction/intro-config-use-fd.adoc

```diff
@@ -0,0 +1,142 @@
+. Optionally configure a reconnection strategy by expanding the *Advanced* tab and completing the fields in the *Connection* section.
++
+For example, this reconnection strategy does the following:
++
+* Forces the deployment to fail if the connection fails and configured reconnection attempts are exhausted
+* Limits the number of reconnection attempts to three at a frequency of every 5000 milliseconds (ms)
++
+.Use the fields in the Reconnection Strategy section to specify reconnection options
+image::introduction/intro-config-overview-reconnection-strategy-fd.png[Reconnection strategy example using Flow Designer]
+. To use TLS, scroll down and specify the required information in the *TLS Configuration* section.
+. Click *Save*.
+
+For more information about configuring `HTTP Listener`, see the xref:http/http-documentation.adoc[HTTP Connector Reference].
+
+[[scheduler]]
+=== Use Scheduler as an Input Source
+
+Use `Scheduler` to trigger a flow when a time-based condition is met. You can configure `Scheduler` to trigger a flow at a regular interval or give it a more flexible cron expression.
+
+To use `Scheduler` as an input source:
+
+. Follow the procedure described in <<configure-input-source,Configure an Input Source for the Flow>>.
+. In the *Search* field, enter `Scheduler`.
+. Select *Scheduler*.
+. In the *Scheduling Strategy* field, select whether you want to use a fixed frequency or cron expression to trigger the flow:
++
+For example, the following configuration directs `Scheduler` to:
+
+* Wait 10 seconds before initiating the flow for the first time after the app starts
+* Initiate the flow every 30 seconds after the initial 10 second delay
++
+.Select `Fixed Frequency` in the *Scheduling Strategy* field to initiate a flow at regular intervals.
+image::introduction/intro-config-fd-scheduler-fixed.png[Configuration for a fixed scheduling strategy]
++
+
+The following configuration uses a cron expression to log `my message` every second:
++
+.Select `Cron` in the *Scheduling Strategy* field to use a cron expression to specify when to initiate a flow.
+image::introduction/intro-config-fd-scheduler-cron.png[Configuration using a cron expression]
+
+For more information about configuring `Scheduler`, see xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint]
+
+
+[[connector-specific]]
+=== Use a Connector-Specific Input Source
+
+If a connector has associated input sources, you can use a connector input source to initiate the flow. For example, you can use `Amazon SQS Receive Messages` as an input source for Anypoint connector for Amazon SQS (Amazon SQS Connector). This input source initiates a flow each time the configured Amazon SQS queue receives a new message.
+
+To start the configuration for connector-specific input sources, follow these steps:
+
+. Follow the procedure described in <<configure-input-source,Configure an Input Source for the Flow>>.
+. In the *Trigger* field, search for and select the connector.
```

Comment:
What Trigger field? Do you mean the one in the wizard that you told me to click out of? On the canvas, there is a box that is labelled "Trigger". Is that what you mean?

Reply:
Robert is correct here. If you use the wizard, there is a Trigger field where you can search for the trigger. But if you follow the procedure you are linking to, where we are telling them to go straight to the canvas, then it's a Trigger card on the canvas.